modules/apps/frontend-css/frontend-css-test/src/testFunctional/tests/Cadmin.testcase

```diff
@@ -44,6 +44,8 @@ definition {
 	@description = "LPS-146034. Verifies admin component displays it's default theme after applying a theme"
 	@priority = "5"
 	test CanDisplayDefaultTheme {
+		property portal.acceptance = "true";
+
```

Comment:
Realized I forgot to add the tests to acceptance... but both tests are [upstream](https://testray.liferay.com/home/-/testray/cases/1609092542) now :)